CONTRIBUTING.md

```diff
@@ -31,3 +31,9 @@
 1. Update `CHANGELOG.md`.
    - Add the new section with the release version in the title.
    - Leave the `main` section empty (without any changelog entries).
+
+1. If `[EAS BUILD API]` tag is present in CHANGELOG
+   - Find version of the package `@expo/eas-build-job` used in the eas-cli.
+   - Find `Publish` commit in [expo/eas-build](https://github.com/expo/eas-build/commit/) for that version of `@expo/eas-build-job`. In most cases it will be in the latest publish commit.
```

Comment:
```suggestion
   - Find the `Publish` commit in [expo/eas-build](https://github.com/expo/eas-build/commit/) for that version. In most cases, it will be the latest `Publish` commit.
```